tests/functional/run-s3-tests.sh

```diff
@@ -13,6 +13,7 @@ configure_s3cmd
 RET=0
 
 run_functional_test s3-container-hierarchy.cfg s3_container_hierarchy_v2.sh
+run_functional_test s3-container-hierarchy-key-v2.cfg s3_container_hierarchy_v2.sh
```

Comment:
To be done later: Icing on the cake, we could add the migration script here and check that all objects are available after migrating `redis`

Reply:
I've add some tests.